flags.go

```diff
@@ -23,6 +24,7 @@ func init() {
 	flag.StringVar(&BUILD, "build", "", "Build all pages as static site in this directory")
 	flag.StringVar(&SITENAME, "sitename", "XLOG", "Site name is the name that appears on the header beside the logo and in the title tag")
 	flag.StringVar(&INDEX, "index", "index", "Index file name used as home page")
+	flag.StringVar(&NOT_FOUND_PAGE, "notfoundpage", "404", "Custom not found page")
```

Comment:
what happens if the user doesn't have a 404.md file in their directory?

Reply:
We will make the default value is empty string to avoid this case.

Reply:
Do you think add flag for that is useful for this case, or we make this automatically for 404.md ?

Reply:
if 404.md doesn't exists will this throw an error?

Reply:
No, It will not throw an error i tested it multiple time, because of the line 63 build.go file.